guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -66,6 +66,11 @@ ifndef::foreman-deb[]
 include::topics/upgrading_virt_who.adoc[leveloffset=+2]
 endif::[]
 
+ifdef::satellite[]
+// Migrating to a Newly Deployed Satellite Server
+include::common/modules/proc_migrating-to-a-new-server.adoc[leveloffset=+2]
+endif::[]
```

Comment:
[This](https://github.com/theforeman/foreman-documentation/pull/1311#discussion_r899870489) was marked as resolved (and indeed, the section was moved), but I don't think the new placement is any better?

The virt-who section in between seems odd? And there is no hint that this is an alternative to the already documented 7→8 upgrade.

Reply:
Moved and added some other items to master.adoc.